whitepaper/whitepaper.md

```diff
@@ -284,32 +284,31 @@ gets replaced by a traditional bank account holding USD.
 
 
 ---
 Motivation: The suitability of a blockchain solution regarding a use case depends on the underlying blockchain properties.
             As blockchain technologies are maturing at a fast pace, in particular private blockchains, its properties might change. Consequently, this creates an unbalance between user expectations' and the applicability of the solution.
             It is, therefore, desirable for an organization to be able to replace the blockchain providing the infrastructure to a certain service.
           
-Currently, when a consortium wants to migrate their blockchain (e.g., the source blockchain became obsolete, cryptographic algorithms no longer secure, etc), the solution is to re-implement business logic using a different platform, yielding great effort. Migrating data to another blockchain currently is not feasible in practice. 
-This is one of the reasons mass adoption amongst enterprises grows slowly.
-Data migrations have been performed by (Frauenthaler et al., 2019) and (Scheid et al., 2019), both recent endeavors to render flexibility to blockchain-based solutions.
-In those works, the authors propose simple data migration capabilities for public, permissionless blockchains, in which a user can specify requirements for its blockchain. The developed solutions allow "switchovers," where a blockchain is migrated, in case there is a blockchain that better satisfies the current requirements.
-Nonetheless, arguably a more interesting approach would be to consider cross-smart contract execution functionality or another automatic way of migration.
 
-References: <br>
-E Scheid and Burkhard Rodrigues, B Stiller. 2019. Toward a policy-based blockchain agnostic framework. 16th IFIP/IEEE International Symposium on Integrated Network Management (IM 2019) (2019)
-Philipp Frauenthaler, Michael Borkowski, and Stefan Schulte. 2019. A Framework for Blockchain Interoperability and Runtime Selection.
+Currently, when a consortium wants to migrate their blockchain (e.g., the source blockchain became obsolete, cryptographic algorithms no longer secure, etc), the solution is to re-implement business logic using a different platform, yielding great effort. 
+Data migrations have been performed before on public blockchains [2,3], both recent endeavors to render flexibility to blockchain-based solutions.
```

Comment:
Can create links that navigate to the references section upon a click when previewing the markdown.
Example links for the 2 and 3 could look like this:

`<sup>[2](#7-references)</sup>`

`<sup>[3](#7-references)</sup>`